app/controllers/projects_controller.rb

```diff
@@ -3,10 +3,32 @@ class ProjectsController < ApplicationController
   before_action :login_required, only: [:new]
 
   def new
+    @project = Project.new
   end
 
   def index
     @projects = Project.all
   end
 
+  def create
+    @project = Project.new(project_params)
+
+    @project.submitter_id = current_user.id
```

Comment:
Short answer: I don't think we need to. 

Longer answer: I would really love to see squashed commits when there was a lot of noise due to some exploration – for instance when students are working on the app. But squashing commits is scary and no student should be bothered with it, hence: nope, let's not squash commit. :)